rdrf/rdrf/helpers/utils.py

```diff
@@ -705,6 +705,31 @@ def get_supported_languages():
     return [Language(pair[0], pair[1]) for pair in settings.LANGUAGES]
 
 
+def get_all_language_codes():
```

Comment:
This function has a quadratic time complexity because of how it iterates over the list of languages. Need to refactor it to be more efficient.

The loop can probably be replaced by something that builds the list in three stages. You could:

1. Optionally add the `pseudo` language first (or last), rather than checking each iteration in a loop.
2. Extend the list with only the languages_in_settings
3. Extend the list with the remaining languages

Also, the list should probably be sorted by language code, or whatever the standard way of sorting languages is.